src/main/java/duke/AddDeadlineCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Handles addition of deadline-based Tasks.
+ */
+public class AddDeadlineCommand extends Command {
```

Comment:
`AddDeadlineCommand`, `AddEventCommand` and `AddTodoCommand` are very similar. Perhaps can combine these 3 classes to `AddTaskCommand`.